package.json

```
@@ -9,9 +9,11 @@
     "test:demo:browser": "yarn --cwd examples/express && yarn --cwd examples/express test",
     "test:demo:nodejs": "node examples/node/index.js",
     "test:demo:esm": "yarn --cwd examples/esm && yarn --cwd examples/esm start",
+    "test:playground:vue": "yarn --cwd ./playgrounds/vue && yarn --cwd ./playgrounds/vue build",
```

It'snot really a test, but a build:

```suggestion
    "build:playground:vue": "yarn --cwd ./playgrounds/vue && yarn --cwd ./playgrounds/vue build",
```